pkg/backend/oss.go

```diff
@@ -63,114 +61,104 @@ func newOSSBackend(rawConfig []byte) (*OSSBackend, error) {
 	return &OSSBackend{
 		objectPrefix: objectPrefix,
 		bucket:       bucket,
+		forcePush:    forcePush,
 	}, nil
 }
 
-// Ported from https://github.com/aliyun/aliyun-oss-go-sdk/blob/c82fb81e272d84f716d3f13c36fe0542a49adfeb/oss/utils.go#L207.
-func splitBlobByPartNum(blobSize int64, chunkNum int) ([]oss.FileChunk, error) {
-	if chunkNum <= 0 || chunkNum > 10000 {
-		return nil, errors.New("chunkNum invalid")
+func splitFileByPartSize(blobSize, chunkSize int64) ([]oss.FileChunk, error) {
+	if chunkSize <= 0 {
+		return nil, errors.New("invalid chunk size")
 	}
 
-	if int64(chunkNum) > blobSize {
-		return nil, errors.New("oss: chunkNum invalid")
+	var chunkN = blobSize / chunkSize
```

Comment:
It will leak the last chunk

Reply:
It will be appended later.

Reply:
Can we have a comment telling the trick at least?

Reply:
It's ported from https://github.com/aliyun/aliyun-oss-go-sdk/blob/v2.2.6/oss/utils.go#L259, leave the port code as unchanged as possible.